trinity/protocol/eth/managers.py

```diff
@@ -56,5 +68,41 @@ class GetBlockHeadersRequestManager(BaseRequestManager):
     def _send_sub_proto_request(self, request: HeaderRequest) -> None:
         self._peer.sub_proto.send_get_block_headers(request)
 
-    def _normalize_response(self, response: Tuple[BlockHeader, ...]) -> Tuple[BlockHeader, ...]:
-        return response
+    async def _normalize_response(self,
+                                  msg: Tuple[BlockHeader, ...]
+                                  ) -> Tuple[BlockHeader, ...]:
```

Comment:
I find that style harder to read than this one.  Thoughts?

Reply:
Yeah, it comes up a *lot* more with all the type boilerplate.

My biggest issue is that it means the intend is tied to the length of the method name, so changing the method name:
- triggers a much larger diff than necessary
- takes more work to re-indent the args, depending on your local IDE setup

Also, for long method names, it is awkward to read variables dangling way off to the right. It might not even avoid the line width limit that well.

Reply:
I personally like the `self` staying on the top line but then again I think we may soon migrate to something like black anyway ;)